test/index.js

```diff
@@ -1955,4 +1956,35 @@ describe('Toys', () => {
             expect(Toys.asyncStorageInternals()).to.be.an.instanceof(Map);
         });
     });
+
+    describe('joi patch schema', () => {
```

Comment:
Just for consistency with the rest of the `describe()` blocks:
```suggestion
    describe('patchJoiSchema()', () => {
```